products/compute/api.yaml

```
@@ -1285,6 +1285,67 @@ objects:
           An optional textual description of the valid disk size, such as
           "10GB-10TB".
         output: true
+  - !ruby/object:Api::Resource
+    name: 'DiskResourcePolicyAttachment'
+    input: true
+    base_url: projects/{{project}}/zones/{{zone}}/disks
```

```suggestion
    base_url: projects/{{project}}/zones/{{zone}}/disks/{{disk}}
```

We build the import format off the `base_url` parts + `identity` values.

We're constructing [`{{project}}/{{zone}}/{{name}}`](https://github.com/terraform-providers/terraform-provider-google/pull/4409/files#diff-7741ec9e30f8047d04317b3c87911d74R236) right now, but we need `{{disk}}` as well for the import fn to expect the id format of `{{project}}/{{zone}}/{{disk}}/{{name}}`.

I don't think anywhere will use the `base_url`, so this should be a safe change. If you find this breaks something else, we can explicit set the `import_id` beside `id_format`.